zerver/lib/bugdown/__init__.py

```diff
@@ -1552,9 +1566,9 @@ def bugdown_stats_finish():
     bugdown_total_requests += 1
     bugdown_total_time += (time.time() - bugdown_time_start)
 
-def convert(content, message=None, message_realm=None, possible_words=None, sent_by_bot=False):
-    # type: (Text, Optional[Message], Optional[Realm], Optional[Set[Text]], Optional[bool]) -> Text
+def convert(content, message=None, message_realm=None, possible_words=None, slash_commands=None, sent_by_bot=False):
```

Comment:
I find it slightly confusing that `slash_commands` refers to the set of all available slash commands here, while the same name `slash_commands` on a message object refers to the set of all triggered slash commands for the message.

Would it make sense to do one or both of the following?
- Rename the function parameter here from `slash_commands` to `available_slash_commands`
- Rename the field on the message object from `slash_commands` to `triggered_slash_commands`